plugins/scaffolder-backend-module-sentry/src/actions/createProject.ts

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { createTemplateAction } from '@backstage/plugin-scaffolder-backend';
+import { InputError } from '@backstage/errors';
+import { Config } from '@backstage/config';
+
+/**
+ * Creates the `sentry:craete-project` Scaffolder action.
+ *
+ * @remarks
+ *
+ * See {@link https://backstage.io/docs/features/software-templates/writing-custom-actions}.
+ *
+ * @param options - Configuration of the Sentry API.
+ * @public
+ */
+export function createSentryCreateProjectAction(options: { config: Config }) {
+  const { config } = options;
+
+  return createTemplateAction<{
+    organizationSlug: string;
+    teamSlug: string;
+    name: string;
+    slug: string;
+    authToken?: string;
+  }>({
+    id: 'sentry:create-project',
```

Comment:
I wonder if this should be something like `sentry:project:create` as we have `github:repo:create` already although no real strong opinion here to be honest.

Reply:
makes sense, also if we add more sentry scaffolder actions we have a clear namespaces for the actions.

is done.